concrete/docs/SUMMARY.md

```diff
@@ -24,7 +24,7 @@
 * [Dynamic Types](how\_to/dynamic\_types.md)
 * [Generic Bounds](how\_to/generic\_bounds.md)
 
-## Developer
+## Developers
```

Comment:
in the readme we have a link to developers/contributing and I chose to keep the plural form